mantidproject/mantid
# Cancel button frequency domain analysis

This adds a cancel button to the MaxEnt widget of the Frequency domain analysis GUI (in muon section). 

**To test:**
1. Open muon analysis and load some data (the data does not matter). Close muon analysis
2. Open Frequency Domain Analysis and select the MaxEnt option from the combo box at the top of the window
3. Run the algorithm, wait a little while then press cancel.
4. The algorithm should stop (the progress bar in the algorithm dialog will disappear) and the calculate button will be enabled
5. Run maxEnt again, this time instead of pressing cancel close the frequency domain analysis window and see if it stops the algorithm (it should)




Fixes #20879. 

**Release Notes** 

Added to release notes 
---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
